src/js/view/view.js

```diff
@@ -41,44 +40,19 @@ let ControlsModule;
 const _isMobile = OS.mobile;
 const _isIE = Browser.ie;
 
-function View(_api, _model) {
+function View(_api, _model, _captionsRenderer) {
```

Comment:
Can we change `_model` to `_viewModel` in the view for disambiguation?

Reply:
I avoided doing this because model in all files under view/ (except for view-model) refer to the viewModel, and I want the initial diff for this to be as small as possible.